dotcom-rendering/src/palette.ts

```diff
@@ -82,6 +239,14 @@ const paletteColours = {
 		light: headlineBackgroundColourLight,
 		dark: headlineBackgroundColourDark,
 	},
+	'--background-avatar': {
+		light: backgroundAvatarLight,
+		dark: backgroundAvatarDark,
+	},
+	'--background-article': {
+		light: backgroundArticleLight,
+		dark: backgroundArticleDark,
+	},
```

Comment:
I followed the pattern established in AR of splitting specific colour functions by type such as “text”, “backround”, “line”, etc.

I can see that as this approach is not nested, it may be more accurate to organise things by themes such as “star”, “avatar”, “card”, etc.

Emoji react: 🎉 type / 🚀 themes

Reply:
Naming ≈ Organising 😉